vaadin-app-layout-flow-parent/vaadin-app-layout-flow-integration-tests/src/main/java/com/vaadin/flow/component/applayout/examples/I18nPage.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
```

Comment:
Nit: Could update the copyright year, same in other added files

Reply:
Fixed.